Modules/Filtering/DisplacementField/test/itkTimeVaryingVelocityFieldIntegrationImageFilterTest.cxx

```diff
@@ -123,7 +123,7 @@ itkTimeVaryingVelocityFieldIntegrationImageFilterTest(int, char *[])
   size[0] = 3;
   size[1] = 3;
   size[2] = 401;
-  size[3] = 41;
+  size[3] = 61;
```

Comment:
Commit message should be updated to say "This a a modification to the time-varying test".

Reply:
I'm not sure how to change this message now.

But I could merge the commit with the previous one. Do you think this is a good idea?

Reply:
Here are some explanations of how to change a commit message:
https://stackoverflow.com/questions/1884474/change-old-commit-message-using-git-rebase
If you use some GUI tool such as TortoiseGit, it is even easier.

It is better not to merge, as making time normalization optional, and exercising that option are two different things, and it is best to have them as separate commits.

Reply:
Thanks for the pointer. I think that I have resolved it now. Let me know if it looks correct from your side.

Reply:
You had two branches (one with original message, one with updated message) which were merged at the end, producing a strange looking history when linarized by GitHub. I picked the correct one and pushed it your branch. It is ready for merging now. Let's wait until CI finishes before doing that.

Reply:
Thanks for solving that merge.